<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mapTab">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QWidget" name="mapLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="mapGridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="keyLabel">
         <property name="text">
          <string>Key:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="keySpinBox"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="valueSpinBox"/>
       </item>
       <item row="1" column="0" rowspan="5">
        <widget class="QTableWidget" name="mapTableWidget">
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="mapInsertButton">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="mapRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mapContainsButton">
         <property name="text">
          <string>Contains</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="mapResultLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="mapClearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="setTab">
     <attribute name="title">
      <string>Set</string>
     </attribute>
     <widget class="QWidget" name="setLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="setGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="insertElementLabel">
         <property name="text">
          <string>Insert Element:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="setElementSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="setInsertButton">
         <property name="text">
          <string>Insert</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="setRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="setContainsButton">
         <property name="text">
          <string>Contains</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="setResultLabel">
         <property name="text"/>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="setClearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="setElementsLabel">
         <property name="text"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
